### DISCUSSION TITLE
PAC PAckage command is throwing error " Config File Missing"

### DISCUSSION THREAD

**Discussion by SachinGoyalGL** [microsoft/powerplatform-build-tools#797] [Q&A]
I am using " Program Registration" package generated for Power Pages templates project.
When i try to deploy it using .zip file or dll file, I get an error saying 'Config file missing"

I can see importconfig.xml file is already present in the package zip folder.

![image](https://github.com/microsoft/powerplatform-build-tools/assets/100751504/39655d4a-399f-4d9a-bb02-d2bc37bb5451)

![image](https://github.com/microsoft/powerplatform-build-tools/assets/100751504/9bd71e9c-047a-4261-a56d-40fd2a968f56)

**Comment by v-nbhalerao** [comment]
FYI @ashishchoudhary001 @petrochuk

**Comment by zekelinAlex** [comment]
Did you solve it somehow?

**Reply by Atugonza837** [reply]
One file is missing
